### PR TITLE
Added the Queue trigger for C# and fixed the Queue trigger for node

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: Templates/QueueTrigger-CSharp/function.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿{
+    "bindings": [
+        {
+            "name": "myQueue",
```
Comment (fabiocav) [inline Templates/QueueTrigger-CSharp/function.json]: Use "myQueueItem" ?

File: Templates/QueueTrigger/function.json [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
-      "type": "blob",
-      "name": "receipt",
-      "direction": "out",
-      "path": "samples-workitems/{id}"
-    }
-  ]
-}
+    "bindings": [
+        {
+            "name": "myQueue",
```
Comment (fabiocav) [inline Templates/QueueTrigger/function.json]: Same here ("myQueueItem")

**Comment by paulbatum** [comment]
I agree with Fabio's feedback - update the param names as suggested and I'll merge.

**Comment by soninaren** [comment]
@paulbatum have updated the parameter names

**Comment by fabiocav** [comment]
:+1:

File: Templates/QueueTrigger/readme.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# QueueTrigger - C<span>#</span>
+
+The `QueueTrigger` makes it incredibly easy to react to new Queues inside of Azure Queue Storage. This sample demostrates a simple use case of processing data from a given Queue using C#.
```
Comment (mathewc) [inline Templates/QueueTrigger/readme.md]: Not "new Queues", "new Queue messages"

File: Templates/QueueTrigger/index.js [hunk 4]
```diff
@@ -1,5 +1,4 @@
-﻿module.exports = function (workItem, context) {
-    context.log('Node.js queue trigger function processed work item ' + workItem.id);
-
-    context.done(null, workItem);
+﻿module.exports = function (myQueueItem, context) {
+    context.log('Node.js queue trigger function processed work item ' + myQueueItem.id);
```
Comment (mathewc) [inline Templates/QueueTrigger/index.js]: nit: the "work item" nomenclature is a carry over from my earliest samples. Looks like you've generalized the samples, so perhaps just "processed item" or "processed message" makes more sense?

File: Templates/QueueTrigger-CSharp/readme.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# QueueTrigger - C<span>#</span>
+
+The `QueueTrigger` makes it incredibly easy to react to new Queues inside of Azure Queue Storage. This sample demostrates a simple use case of processing data from a given Queue using C#.
```
Comment (mathewc) [inline Templates/QueueTrigger-CSharp/readme.md]: Not "new Queues" - it should be "new Queue messages"

**Pull request merged**